bitrise.yml

```diff
@@ -3,121 +3,142 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 app:
   envs:
-  - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-flutter-ios-android.git
-  - ORIGIN_SOURCE_DIR: $BITRISE_SOURCE_DIR
+    - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-flutter-ios-android.git
+    - SAMPLE_APP_BRANCH: "manual"
+    - ORIGIN_SOURCE_DIR: $BITRISE_SOURCE_DIR
+    - PROJECT_LOCATION: "$BITRISE_SOURCE_DIR/_tmp"
+    - GENERATE_CODE_COVERAGE_FILES: "no"
 
-  
 workflows:
   ci:
     before_run:
-    - audit-this-step
+      - audit-this-step
     steps:
-    - go-list:
-    - golint:
-    - errcheck:
-    - go-test:
+      - go-list:
+      - golint:
+      - errcheck:
+      - go-test:
     after_run:
-    - test
+      - test
 
   test:
+    title: Run E2E tests
+    after_run:
+      - test-without-codecoverage
+      - test-with-codecoverage
+      - test-not-root-project-with-codecoverage
+
+  common:
+    after_run:
+      - assert-output
+      - validate-coverage
+    title: Prepare test
+    steps:
+      - script:
+          title: Delete _tmp
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
+                cd ${ORIGIN_SOURCE_DIR}
+                rm -rf "./_tmp"
+                mkdir "_tmp"
+      - change-workdir:
+          title: Switch working dir to test / _tmp dir
+          description: |-
+            To prevent step testing issues, like referencing relative
+            files with just './some-file' in the step's code, which would
+            work for testing the step from this directory directly
+            but would break if the step is included in another `bitrise.yml`.
+          run_if: true
+          inputs:
+            - path: ${ORIGIN_SOURCE_DIR}/_tmp
+            - is_create_path: true
+      - script:
```

Comment:
pls use the simple-git-clone step: https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-application-loader/blob/master/bitrise.yml#L114